modules/ebi-metagenomics/samtools/bam2fq/main.nf

```diff
@@ -21,7 +21,7 @@ process SAMTOOLS_BAM2FQ {
 
     script:
     def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    def prefix = task.ext.prefix ? "decontaminated_${meta.id}": "${meta.id}"
```

Comment:
To keep things consistent, I would put this one in the `task.ext` in the modules just like BWA_MEM2 (https://github.com/EBI-Metagenomics/miassembler/pull/6/files#diff-bf809c928cb10b54d251dec8a140a2d5505150f97175d8d7b51dda9cc57971feR88)